app/app.controllers.js

```diff
@@ -17,4 +22,23 @@ exports.getAllTopics = ( req,res,next)=>{
      selectTopics().then((topics)=>{
         res.status(200).send({topics});
     }).catch(next)
-}
\ No newline at end of file
+}
+
+
+exports.getArticle = (req, res, next) => {
+
+
+  if (Object.keys(req.params).length === 0) {
+selectArticle()
+      .then((articles) => {
+        res.status(200).send({ articles });
+      })
+      .catch(next);
```

Comment:
Please see my comment in the app.js file. There should be a separate controller for this.